packer/windows/conf/bin/bk-install-elastic-stack.ps1

```diff
@@ -86,7 +87,7 @@ timestamp-lines=${Env:BUILDKITE_AGENT_TIMESTAMP_LINES}
 hooks-path="C:\buildkite-agent\hooks"
 build-path="C:\buildkite-agent\builds"
 plugins-path="C:\buildkite-agent\plugins"
-git-mirrors-path="C:\buildkite-agent\git-mirrors"
+git-mirrors-path="${Env:BUILDKITE_AGENT_GIT_MIRRORS_PATH}"
```

Comment:
I don't know much Windows / PowerShell and was worried this might fail if the env wasn't set at all, which is the case when `BUILDKITE_AGENT_ENABLE_GIT_MIRRORS_EXPERIMENT` is false.

I've confirmed¹ that it results in `git-mirrors-path=""` in the Windows `buildkite-agent.cfg` file when that env var isn't present 👌🏼

¹ a decade of using AWS/EC2, and this is the first time I've ever connected to a windows instance 😬